### PR TITLE
Candid Health Sample Integration

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: src/examples/candid-health/send-to-candid.test.ts [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+      address: [
+        {
+          line: ['575 BEECH STREET'],
+          city: 'HOLYOKE',
+          state: 'MA',
+          postalCode: '01040-2223',
+          country: 'US',
+        },
+      ],
+      resourceType: 'Organization',
```
Comment (reshmakh) [inline src/examples/candid-health/send-to-candid.test.ts:77]: Move this line to 67 for legibility

File: src/examples/candid-health/send-to-candid.ts [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+} from '@medplum/fhirtypes';
+import fetch from 'node-fetch';
+
+export async function handler(medplum: MedplumClient, event: BotEvent<Encounter>): Promise<any> {
+  const encounter = event.input;
+
+  // Read the Patient
+  const patient: Patient = await medplum.readReference(encounter.subject as Reference<Patient>);
+
+  // Encounter.serviceProvider represents rhe organization that is primarily responsible for this Encounter's services
```
Comment (reshmakh) [inline src/examples/candid-health/send-to-candid.ts:20]: typo "the organization"

File: src/examples/candid-health/send-to-candid.ts [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+ * Authenticates into the Candid Health API using API key and API secret, and posts the CodedEncounter object to
+ * Candid's /v1/coded_encounters endpoint
+ * @param candidCodedEncounter A JS representation of the CodedEncounter object
+ * @param apiKey  Candid Health API Key
+ * @param apiSecret Candid Health API Secret
+ * @returns
+ */
+async function submitCandidEncounter(candidCodedEncounter: any, apiKey: string, apiSecret: string): Promise<any> {
+  // Get a Bearer Token
+  const authResponse = await fetch('https://api-staging.joincandidhealth.com/api/v1/auth/token', {
```
Comment (reshmakh) [inline src/examples/candid-health/send-to-candid.ts:123]: maybe move this to a constant like CANDID_BASE_URL or something like that and pass it into the function?

e.g. 
`const authResponse = await fetch(CANDID_BASE_URL + '/api/v1/auth/token'...`

File: src/examples/candid-health/send-to-candid.ts [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+    method: 'post',
+    body: JSON.stringify({
+      client_id: apiKey,
+      client_secret: apiSecret,
+    }),
+    headers: { 'Content-Type': 'application/json' },
+  });
+
+  const bearerToken = ((await authResponse.json()) as any).access_token;
```
Comment (reshmakh) [inline src/examples/candid-health/send-to-candid.ts:132]: Pretty straightforward - like it.

File: src/examples/candid-health/send-to-candid.test.ts [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+    });
+
+    const encounter = await medplum.createResource({
```
Comment (reshmakh) [inline src/examples/candid-health/send-to-candid.test.ts:80]: Is this from some kind of IG?  Probably good to link to it if so?

Comment (rahul1) [reply src/examples/candid-health/send-to-candid.test.ts:80]: Acutally no - this was hand-crafted

**Review by codyebberson — APPROVED** [review]

**Pull request merged**